cloud/services/object_storage_buckets.go

```diff
@@ -31,7 +29,7 @@ func EnsureObjectStorageBucket(ctx context.Context, bScope *scope.ObjectStorageB
 	opts := linodego.ObjectStorageBucketCreateOptions{
 		Region: bScope.Bucket.Spec.Region,
 		Label:  bScope.Bucket.Name,
-		ACL:    linodego.ACLPrivate,
+		ACL:    linodego.ObjectStorageACL(bScope.Bucket.Spec.ACL),
```

Comment:
Question: Where is the cors var being set? I'm assuming we have to set before calling Linode API for obj creation?

Reply:
Hey, I had a doubt about his, these changes have not been pushed yet.
Linodego API defines CorsEnabled *bool , so for CAPL in LinodeObjectStorageBucketSpec should the field be defined as 
`CorsEnabled bool` and coverted as following `opts := linodego.ObjectStorageBucketCreateOptions{
		Region:      bScope.Bucket.Spec.Region,
		Label:       bScope.Bucket.Name,
		ACL:         linodego.ObjectStorageACL(bScope.Bucket.Spec.ACL),
		CorsEnabled: util.Pointer(bScope.Bucket.Spec.CorsEnabled),
	}`

OR

directly defined as `CorsEnabled *bool`?

Reply:
I would make `CorsEnabled` a bool pointer personally. Like you have in your second option!

I'm not sure if there is a preferred way in the team. Might be a good question to ask and get opinions.

Reply:
I agree with @komer3 your second solution looks like the correct one